.gitignore

```diff
@@ -82,8 +82,4 @@ tags
 operator-sdk-*
 tmp
 manager
-
-# OLM
-bundle
-config/manifests
```

Comment:
didn't we just add these to .gitignore? do we care about these again?

Reply:
Yes, sorry. I had to make changes to those files and I think they should be in the repo.